dashboard/modules/log/tests/test_log.py

```diff
@@ -136,7 +136,13 @@ def write_log(s):
 
 @pytest.mark.parametrize(
     "test_file",
-    ["test.log", "test#1234.log"],
+    [
+        "test.log",
+        "test#1234.log",
+        "test_file_no_suffix",
+        "test_file_not_register_mimetypes.json",
+        "test_file_not_register_mimetypes.yaml",
```

Comment:
Just confirming, do these files have the `application/octet-stream` content type?

Reply:
yeah, for `*.yaml` but not for `*.json` (which is `application/json`)